### DISCUSSION TITLE
how to run a node

### DISCUSSION THREAD

**Discussion by ahmadiliyasuzakari** [FuelLabs/sway#4738] [General]
do you know, how to run a node? is very easy follow me lets dive in....

**Comment by meliewww** [comment]
interested too

**Comment by CryptoMania666** [comment]
interested too

**Reply by ahmadiliyasuzakari** [reply]
good thread is coming soon

**Comment by ljq888** [comment]
interested too

**Comment by michaelan1118** [comment]
> do you know, how to run a node? is very easy follow me lets dive in....

Me too

**Reply by Caltriz** [reply]
> me too

**Comment by vapsvapsov** [comment]
also very interested in that topic, i dont have any idea how but i really want to know how!

**Comment by Curraso** [comment]
interested too

**Comment by Nar035** [comment]
interested lets make a group

**Comment by PtitGibus80** [comment]
interested too

**Comment by Yasiroto4** [comment]
Interested to learn too

**Comment by Agriman0101** [comment]
Interested to learn
